integration_tests/models/materialized_models/fact_orders.sql

```diff
@@ -1,4 +1,4 @@
 select 
     *
-    ,round(order_total - (order_total/2)) as discount_total
+    ,1 as discount_total
```

Comment:
❓ this is weird, shouldn't it be a percentage or something? I just spent a bunch of time digging through the files trying to work out why the average discount was always 1 in your expectation seed. 

The thing that worries me here is that if your source data isn't that varied then you don't actually get the benefit of testing the aggregations - what if something has gone horribly wrong and you're actually calculating max/min or something?

Reply:
Very good point - I was trying to keep it as simple as possible but if it doesn't represent the actual calculation then thats not helpful

Reply:
Oh actually this is still a holdover from integration testing! These will mainly just be used for local development as we move the CI testing to pytest to test functionality. So the old version with round is retained in the pytest models 👯